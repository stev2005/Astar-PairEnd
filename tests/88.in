CGGAAGAGCTTCCCATTTTATCCTCCTGAACCGTGGCGCGCATTCTGAGCCGGGCTTACTAGCGGTCTCGGTGCGTCAATCCCTTGGTGTGGCTCTGGTAACTGGCCTCCTGGACGTTCCGTATTCTCTGCTTACCCGCGAAAGACATACCGCCGAGCCCTTAGTCTCTATTAAGCC
15
AGACACGGCATGCGCCACGAGGACCCCTGTGGCGACTGAATT
AGAGGCAGTGTAAAGTGCCAGTATGTTGGTCGCCATGCGCTCCTCTTTAATGAAGGTCCCAGTCTACCCTGAATACTTATTTGTTAGGCTTGCTTATACATT
TGAACTACTATGTTAGATCGAAGATGCTGAACTAGAAGCTGACGTGAACCGGGAGCTACCACGTCAACTCACGGTCATCCCATTAGATGCAGGTGATGCGTGACATTATACAACTCTTACCCTTACTTTC
TTGATCTTAAAAACTCATGCATCTTGTGGAA
TGCGACCACCCCTATTTCCTAGTTAGCGGGTCACTAGACTCGAAG
AGACCATATTGCGATCCTGAAAACTGTAGCTGTATCAGCAGACAAACGTAGTATAACTATCTGATCCAATAGTCGTCTCATAAATAAAAATCTCCTTTTTGTCCATACGGCGGCGCGGCGGATCGAACTT
CGCGGATTGCCATGGAAACGGCCATGAACAATGCCCCAAACC
AATAATCTGTAGGGGTAGTTCCAGGCGGGATTAGTACCTTAACGGACGGCCTGGCATATCAGAAAAACCACCAGTGGCCCGGAAAGATGTCGCCGCCGTGGATGGTCCAGGGAGG
AGAGGCATTGCACATTAGGGCTTCCCTCACGAATTGATCTGGATGTGGCACTTAAAGAGGCAGCTCTACCTTTTGGGAAACCTAGTCATTGCTAGTGGTTTGGGGAAAACACGTGATTAGGGTCTGA
ATATGGCAGGAGTCACAGG
CGGTCCAATCCTACGGTAGCGTTCATGACAAGTGCATGATTCGTGCGCCAGTTCATCGAGTACGGGGCAGATTTTCACGCCAACCAAGGACCAGTGACACTAAGTTTTACAGCATTATCTAGAAAATCCTGCTGTTGTGGATAGTTCATGTTCTTC
GGCCAGCTCACAGTCCATCTACGGACTTCCTTT
CAGGTTGAAATCGTATTAACGATGTATGTGGAGAGCAACACCCTACGTCGCTGACCAAATTGACTGTTTATACATGCCCGTGCCGTGGTGGGCGAAATTTTGTCGGTTTACGTGTCCGACAGTCATCAGAAGGTTAGGTC
AACTTTACTCGCCTCACCTCCCACAATGAAAAAGTACACACTCGTGTATGCCTCGTGCCGCCGGTGGAGTATTCGGTCGTTTAGCGCTTGCATTGTCATTAATTCCCC
TACGATTGAGCGTCCTAAAAAAAATTCAAGGTGTGGGCAGACATTCGTGTTGTAGGTTGAGATATCTCTTTTCTGAACTGAGATGGCGGTGATCAGTTTGTGCACTACA
