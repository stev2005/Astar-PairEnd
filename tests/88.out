18
50
64
14
19
65
19
55
62
8
76
13
67
52
54
